hw2/04-background.js

```diff
@@ -1 +1,40 @@
 // Add your code here
+const input = document.querySelector('#interval-input');
+const button = document.querySelector('#start-stop');
+button.classList.add('start');
+const bodyElement = document.querySelector('body');
+const maxValue = 255;
+let intervalID = NaN;
+bodyElement.style.backgroundColor = 'rgb(108 97 219)';
+
+function changeBackgroundColor() {
+  const red = Math.floor(Math.random() * maxValue);
+  const green = Math.floor(Math.random() * maxValue);
+  const blue = Math.floor(Math.random() * maxValue);
+  const randomColor = `rgba(${red}, ${green}, ${blue}, 0.5)`;
+
+  bodyElement.style.backgroundColor = randomColor;
+}
+
+function handleButtonClick() {
+  const interval = input.value;
+  const buttonClasses = button.classList;
+
```

Comment:
You might consider adding a check to make sure the user input is a number. I know you set the field to be type number, but in some browsers this can be ignored and users are able to add letters.

Reply:
whaaaa kinda crap browsers do that lol